web-ng/src/app/components/map/map.component.ts

```diff
@@ -60,6 +57,8 @@ export class MapComponent implements OnInit, AfterViewInit, OnDestroy {
     streetViewControl: false,
     mapTypeId: google.maps.MapTypeId.HYBRID,
   };
+  private focusedMarker: google.maps.Marker | null = null;
```

Comment:
Can we use undefined here instead? Then it would be:
`private focusedMarker?: google.maps.Marker;`

Reply:
I think null makes sense, why undefined?